### DISCUSSION TITLE
Can you add functionality to spawn objects in spesific scene using spawn?

### DISCUSSION THREAD

**Discussion by Stormhound** [FirstGearGames/FishNet#320] [Documentation]
I'm having a bit hard time on spawning object in stacked scenes. Currently I'm using RPCs to tell client to move specific Network Object to target scene. Can you add functionality to spawn objects in spesific scene using spawn?

**Comment by FirstGearGames** [comment]
This is more a documentation bit than feature. I've changed the category and accepted the discussion. I'll create a guide at some point on how to do this.
